content/patterns/radio/examples/radio.html

```diff
@@ -25,120 +25,121 @@
+            <div role="radio" aria-checked="false" tabindex="0">レギュラークラスト</div>
+            <div role="radio" aria-checked="false" tabindex="-1">ディープディッシュ</div>
+            <div role="radio" aria-checked="false" tabindex="-1">薄いクラスト</div>
           </div>
 
           <div role="radiogroup" aria-labelledby="group_label_2" id="rg2">
-            <h3 id="group_label_2">Pizza Delivery</h3>
-            <div role="radio" aria-checked="false" tabindex="0">Pickup</div>
+            <h3 id="group_label_2">ピザの配達</h3>
+            <div role="radio" aria-checked="false" tabindex="0">受け取り</div>
 
-            <div role="radio" aria-checked="false" tabindex="-1">Home Delivery</div>
+            <div role="radio" aria-checked="false" tabindex="-1">宅配</div>
 
-            <div role="radio" aria-checked="false" tabindex="-1">Dine in</div>
+            <div role="radio" aria-checked="false" tabindex="-1">レストランで食事</div>
           </div>
         </div>
-        <div role="separator" id="ex_end_sep" aria-labelledby="ex_end_sep ex_label" aria-label="End of"></div>
+        <div role="separator" id="ex_end_sep" aria-labelledby="ex_label ex_end_sep" aria-label="の終了"></div>
       </section>
 
       <section>
-        <h2>Accessibility Features</h2>
+        <h2>アクセシビリティ機能</h2>
         <ul>
-          <li>Uses CSS attribute selectors for synchronizing <code>aria-checked</code> state with the visual state indicator.</li>
+          <li>CSS 属性セレクタを使用して、<code>aria-checked</code> 状態を視覚的な状態インジケータと同期します。</li>
           <li>
-            Uses CSS <code>:hover</code> and <code>:focus</code> pseudo-classes for styling visual keyboard focus and hover.
+            視覚的なキーボードフォーカスとホバーのスタイル設定には、CSS の <code>:hover</code> 及び <code>:focus</code> 擬似クラスを使用します。
             <ul>
-              <li>The focus indicator encompasses both the radio button and label, making it easier to perceive which option is being chosen.</li>
-              <li>Hover changes background of both the radio button and label, making it easier to perceive that clicking either the label or button will activate the radio button.</li>
-              <li>The cursor is changed to a pointer when hovering over the radio button to help people with visual impairments identify it as an interactive element.</li>
+              <li>フォーカスインジケータはラジオボタンとラベルの両方を囲み、どのオプションが選択されているかを認識しやすくします。</li>
+              <li>ホバーするとラジオボタンとラベルの両方の背景が変化し、ラベル又はボタンのいずれかをクリックするとラジオボタンがアクティブになることを認識しやすくします。</li>
+              <li>視覚障害のある利用者がインタラクティブな要素として識別できるように、ラジオボタンにカーソルを合わせるとカーソルがポインタに変わります。</li>
             </ul>
           </li>
           <li>
-            Because transparent borders are visible on some systems with high contrast enabled, only the focused element has a visible border.
-            When focusable elements are not focused, they have a 0-width border and additional padding equal in width to the border that is used to indicate focus.
+            ハイコントラストが有効になっている一部のシステムでは透明な境界線が表示されるため、フォーカスされた要素のみが可視境界線を持ちます。
```

Comment:
```suggestion
            ハイコントラストが有効になっている一部のシステムでは透明な境界線が可視化されるため、フォーカスされた要素のみが可視境界線を持ちます。
```
activedecendant例で検討している訳注を入れるとよさそうです